README.md

```diff
@@ -296,6 +302,20 @@ Thanks to [Sindre Sorhus](https://github.com/sindresorhus) and
 
 [Apache 2.0](https://github.com/googlechrome/sw-precache/blob/master/LICENSE) © 2015 Google Inc.
 
+Copyright 2015 Google, Inc.
+
+Licensed under the [Apache License, Version 2.0](https://github.com/googlechrome
+/sw-precache/blob/master/LICENSE)] (the "License"); you may not use this file
```

Comment:
The same issue called out in https://github.com/GoogleChrome/sw-toolbox/pull/51 applies here.